quotesbot/settings.py

```diff
@@ -16,75 +14,75 @@
 
 
 # Crawl responsibly by identifying yourself (and your website) on the user-agent
-#USER_AGENT = 'quotesbot (+http://www.yourdomain.com)'
+# USER_AGENT = 'holidaygems (+http://www.yourdomain.com)'
 
 # Obey robots.txt rules
 ROBOTSTXT_OBEY = True
 
 # Configure maximum concurrent requests performed by Scrapy (default: 16)
-#CONCURRENT_REQUESTS = 32
+# CONCURRENT_REQUESTS = 32
```

Comment:
Although not PEP8, I believe the comment spacing in lines like this one is correct for its purpose, which is marking commented out code, as opposed to code comments. I think we need to think of this file as a template for users, and hence leave it like that.

Also, I’m guessing this may be the result of `scrapy startproject` or similar, in which case the file contents should match those generated by Scrapy.